test/system/notes_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'application_system_test_case'
+
+class NoteTest < ApplicationSystemTestCase
+  Capybara.default_max_wait_time = 60
+
+  test 'clicking tabs in notes by author page' do
+  visit 'notes/author/bob'
+
+  find('.nav-link', text: 'Research Notes').click()
+  page.assert_selector('.nav-link active', text: 'Research Notes')
```

Comment:
It's this line that's failing:

https://github.com/publiclab/plots2/pull/9622/checks?check_run_id=2563784322#step:9:82

Is there a reason it wouldn't be finding that css?

Reply:
Hi @jywarren, thanks for the links to resources!   

These are still failing tests because I haven't coded the template nor the partial for tabs yet!  I'm working on it, but I wanted to make sure these tests would make sense first.
I'll consider the changes soon to be introduced in _nav_tabs.html.erb 
https://github.com/publiclab/plots2/pull/9648/commits/862d0e8856fdbbe9a7be671cd2c4b022a266e450
https://github.com/publiclab/plots2/pull/9647/commits/9b3ba131d018a63f92fa7e324f11cd5c753bfb91
Thanks!

Reply:
That makes total sense, thank you! And much appreciated. Love the tests first approach!